daemon/cluster/noderunner.go

```diff
@@ -357,8 +357,10 @@ func (n *nodeRunner) enableReconnectWatcher() {
 	n.cancelReconnect = cancel
 
 	go func() {
+		type timeoutError interface{ Timeout() bool }
```

Comment:
Are we sure we can't use `net.Error` instead to avoid having to duplicate this type? If the behavior ever changes later I'm worried about having little things like this scattered around that would all need to be updated to match eachother. Not the end of the world though.

Reply:
What behavior are we worried about changing?

Reply:
One of the timeout errors changes, then it doesn't get updated everywhere. However, I see why this was done instead of using net.Error now (per your other comment), so this makes sense. Still might be good to pull timeoutError out somewhere just to reduce duplication though.